mbedtls-sys/vendor/library/aesni.c

```diff
@@ -48,6 +48,12 @@
 
 #if defined(MBEDTLS_HAVE_X86_64)
 
+#if defined(MBEDTLS_FORCE_AESNI)
```

Comment:
Oh I thought you were going to do the opposite: Not define this function so it becomes undefined, and then the linker can figure it out. The behavior you have here doesn't match the feature name.

Reply:
Thanks for the code review. I thought of the feature as _forcing AESNI support_ . But now looking back, _Customized AESNI Support_  would be more suiting here. 
I will make the changes and push a patch.

What should I do about padlock? Should we enable it (under our customizations)?

Reply:
I think you asked me this before. Padlock should be treated the same as AESNI